CMakeLists.txt

```diff
@@ -338,9 +369,10 @@ ELSE(CMAKE_TOOLCHAIN_FILE)
   ENDIF($ENV{CRAYPE_VERSION} MATCHES ".")
 
   #------------------------------------
-  # Identify the compiler
+  # Identify the compiler -- I don't get why this COMPILER variable is added --PD
```

Comment:
It is used to pick compiler vendor specific cmake configuration later. Perhaps, it can be simplified.